src/test/java/uk/gov/companieshouse/siccode/api/search/CombinedSicActivitiesRepositoryTest.java

```diff
@@ -0,0 +1,119 @@
+package uk.gov.companieshouse.siccode.api.search;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.collection.IsIterableContainingInAnyOrder.containsInAnyOrder;
+import static org.hamcrest.collection.IsIterableContainingInOrder.contains;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.Arrays;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.data.mongo.DataMongoTest;
+import org.springframework.data.mongodb.core.MongoTemplate;
+import org.springframework.data.mongodb.core.index.TextIndexDefinition;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+import uk.gov.companieshouse.siccode.api.groups.TestType;
+
+/**
+ *  Do integration testing on an embedded Mongo Database (note that this api will be tested
+ *  later in Karate tests against a "real" Mongo database).
+ */
+@Tag(TestType.INTEGRATION)
+@DataMongoTest
+@ExtendWith(SpringExtension.class)
+class CombinedSicActivitiesRepositoryTest {
+
+	@Autowired
+    private CombinedSicActivitiesRepository combinedSicActivitiesRepository;
+
+	@Autowired
+	private MongoTemplate mongoTemplate;
+
+	private final static CombinedSicActivitiesStorageModel BARLEY_FARMING = new CombinedSicActivitiesStorageModel("10", "01110", "Barley Farming",
+	"barley farming", "Growing of cereals except rice, leguminous crops and oil seeds", false);
+
+	private final static CombinedSicActivitiesStorageModel BARLEY_GROWING = new CombinedSicActivitiesStorageModel("15", "01110", "Barley growing",
+	"barley growing", "Growing of cereals except rice, leguminous crops and oil seeds", false);
+
+	private final static CombinedSicActivitiesStorageModel BEAN_GROWING_ORGANIC = new CombinedSicActivitiesStorageModel("17", "01110", "Bean growing organic",
```

Comment:
IDE is still misconfigured. Please fix it to use spaces.
Do you have the CH formatter installed?

Reply:
Corrected